### ISSUE TITLE
Variable Name is different in lab-03-1

### ISSUE THREAD

**Issue by simonjisu** [deeplearningzerotoall/TensorFlow#12]
lab-03-1-Linear-Regression-and-Minimizing-Cost-eager.ipynb

안녕하세요! 승준님, 조금은 사소한건데 Gradient Descent 구현에서

데이터 정의하실때는 `x_data`, `y_data` 를 쓰셨는데 
```
x_data = [1., 2., 3., 4.]
y_data = [1., 3., 5., 7.]
```

밑에서 연산을 수행할 때는 X, Y 를 쓰셨더라구요. 그래서 이전에 선언하셧던 

```
X = np.array([1, 2, 3])
Y = np.array([1, 2, 3])
```

기준으로 코드가 돌아간것 같은데, `x_data`, `y_data` 두 가지 `X`, `Y` 로 바꿔야하는거 아닌가 싶어서 문의드려요~

영상은 현재 코드 기준이라서, 아마 영상 설명란에 x_data, y_data 는 X, Y = [1, 2, 3] 기준이라고 말해야할 것 같습니다.
